person.rb

```diff
@@ -0,0 +1,16 @@
+class Person
+  def initialize(_name, _age, parent_permission: true)
+    @id = Random.rand(1..1000)
+    @name = 'Unknown'
+    @age = 0
+    @parent_permission = parent_permission
+  end
+
+  attr_accessor :id, :name, :age, :parent_permission
+
+  private
+
+  def of_age?
+    @age >= 18
+  end
+end
```

Comment:
Please ensure a public method `can_use_services?` that returns true if a person is of age or has permission from parents is added to your person.rb file.
  
```
    def can_use_services? 
    of_age?() || @parent_permission
  end
```

Reply:
@rotimiazeez Thanks for highlighting them. I will fix